server/flaskr/database/postgres/handlers/meeting_data_handler.py

```
@@ -122,6 +122,17 @@ def update_meeting_agenda(
             params = [(meetingId, theme) for theme in themes]
             cur.executemany(query, params)
 
+    @classmethod
+    def update_meeting_status(cls, meetingId: int, status: str):
```

Ruff (0.15.0)

[warning] Missing return type annotation for classmethod `update_meeting_status`

Add return type annotation: `bool | None`

(ANN206)